AdminWebsite/AdminWebsite.AcceptanceTests/Steps/DashboardSteps.cs

```diff
@@ -21,7 +22,7 @@ public DashboardSteps(Dashboard dashboard, ScenarioContext injectedContext)
         public void ThenBookAVideoHearingPanelIsDisplayed(string panelText)
         {
             var panels = _dashboard.VhPanelTitle();
-            switch (_scenarioContext.Get<string>("User"))
+            switch (_scenarioContext.Get<UserAccount>("User").Role)
```

Comment:
If _scenarioContext.Get<UserAccount>("User") is null, then .Role will throw an exception

Reply:
done for next commit